### ISSUE TITLE
Compiling Model

### ISSUE THREAD

**Issue by logoman04** [bonlime/keras-deeplab-v3-plus#14]
Any recommendations on compiling the model? Optimizers, loss function, etc.?

**Comment by bhack** [comment]
You can find some of the original parameters at https://github.com/tensorflow/models/blob/master/research/deeplab/train.py

**Comment by Zxr1314** [comment]
But I have used tf training style, and I can't restore ckpt file to the keras model ?could u help me please?
